### ISSUE TITLE
Allow users to Remix their own projects?

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-gui#3501]
Discussing who can and can't Remix which projects, @towerofnix brings up some helpful perspective in this comment:
https://github.com/LLK/scratch-gui/issues/3466#issuecomment-432294229

Basically, some Scratch users like to Remix their *own* projects, and have the Remix relationships provide links up and down the tree.

Questions:
1. Should we make this possible in 3.0?
2. If we do make it possible, should it be a standard item in the File menu? Is there any way we can make it possible, but not distracting or confusing to inexperienced users?

**Comment by MasterOfTheTiger** [comment]
What about "save as a copy"?

**Comment by hedgehog125** [comment]
Maybe "save as a copy" could indicate that it's based off of another project by the same person and have a link to it? Like how it works with remixing?

**Comment by benjiwheeler** [comment]
@MasterOfTheTiger, see @towerofnix's comment here: https://github.com/LLK/scratch-gui/issues/3466#issuecomment-432294229

@hedgehog125, that's a great idea! I'm adding it to the issue description!

**Comment by Alzter** [comment]
I agree. Also, we should still have a "See Community" button even if your project is unshared.
